coralproject/talk
# Port Activity dot polling to subscriptions

## What does this PR do?
- https://www.pivotaltracker.com/story/show/154365617
- https://www.pivotaltracker.com/story/show/154366467
- Switch activity dot polling to be based on subscriptions
- Fix bugs in modqueue sorting logic

## How do I test this PR?
- Activity dot should continue to work as before, regardless of
  - whether I'm currently in the modqueue 
  - whether I'm on another view (e.g. community)
- Try emptying the premod and reported queue with another user,
  while being in another view (e.g. community)
